setup.py

```diff
@@ -399,5 +411,8 @@ def get_outputs(self):
     ext_modules=ext_modules,
     cmdclass={"build_ext": TEBuildExtension},
     install_requires=dlfw_install_requires,
+    extras_require={
+        'test': ['pytest',
+                 'tensorflow_datasets'],},
```

Comment:
Note: `onnxruntime` is needed for the PyTorch tests.

Reply:
I'm not sure how to handle this.
I think adding it for all FW isn't great.
I'm creating a "test_pytorch" extra.